huaweicloud/services/as/resource_huaweicloud_as_group.go

```diff
@@ -768,66 +783,71 @@ func resourceASGroupUpdate(ctx context.Context, d *schema.ResourceData, meta int
 	return resourceASGroupRead(ctx, d, meta)
 }
 
+func forceDeleteASGroup(ctx context.Context, asClient *golangsdk.ServiceClient, d *schema.ResourceData) error {
+	if err := groups.ForceDelete(asClient, d.Id()).ExtractErr(); err != nil {
+		return fmt.Errorf("error deleting AS group %s: %s", d.Id(), err)
+	}
+
+	if err := checkASGroupRemoved(ctx, asClient, d.Id(), d.Timeout(schema.TimeoutDelete)); err != nil {
+		return fmt.Errorf("error deleting AS group %s: %s", d.Id(), err)
```

Comment:
```suggestion
		return fmt.Errorf("error deleting AS group %s: %s", d.Id(), err)
```
seems this error message is incorrect and should be describe the waiting errors.

Reply:
I will check the error message in another `PR`.